src/app/main/mocks/user-service-mock.ts

```diff
@@ -69,7 +72,7 @@ export const USERCHANGEROLE: UserRoleModel = {
 };
 
 export const GETUSERPAGEBLE: UserPageableDtoModel = {
-  page: (UserForListDtoModel['1'] = [
+  page: (userForListModelMock['1'] = [
```

Comment:
just move userForListModelMock['1'] = [... code out of export
and then export const GETUSERPAGEBLE: UserPageableDtoModel = {
  page: userForListModelMock....